lib/Github/Api/Issue/Comments.php

```diff
@@ -23,8 +23,8 @@ class Comments extends AbstractApi
      */
     public function configure($bodyType = null)
     {
-        if (!in_array($bodyType, array('raw', 'text', 'html'))) {
-            $bodyType = 'full';
```

Comment:
As I mentioned in the initial PR message and also further down in the last comment I left, I was recognizing that this was a breaking change but that as it was currently implemented here (and in Repository/Comments) the option that you have defaulting currently does not align with the default behavior that is described in the GitHub API documentation.  

I would suggest aligning with the documentation, noting the breaking change in the release and bumping a major version.

If you aren't interested in doing that, then I would rather go through and change all of the places where I set the default value to `raw` and switch them to `full` but then I would suggest that information gets included somewhere that the default behavior of this library does not match the default behavior of the documentation, to avoid any potential for confusion of users of the library.

Let me know your preference and I'll update accordingly.